index.js

```diff
@@ -0,0 +1,29 @@
+import BookCollection from './modules/bookCollection.js';
+import {
+  booksListPage,
+  addBookPage,
+  contactPage,
+} from './modules/pageDispaly.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+/* eslint-disable */
```

Comment:
- [ ] It would be nice if you remove this line 1 because linters help you fix those errors which can break your application 😨 So kindly remove this line and try to fix your linter error manually 👍